### PR TITLE
Add self to Content Security Policy on /content

### PR DESCRIPTION
**Pull request by clrke**

### CODE REVIEW

**Review by ericatallah — APPROVED** [review]

File: src/subcommand/server.rs [hunk 1]
```diff
@@ -823,7 +823,7 @@ impl Server {
     );
     headers.insert(
       header::CONTENT_SECURITY_POLICY,
-      HeaderValue::from_static("default-src 'unsafe-eval' 'unsafe-inline' data:"),
+      HeaderValue::from_static("default-src 'self' 'unsafe-eval' 'unsafe-inline' data:"),
```
Comment (huuep) [inline src/subcommand/server.rs:826]: can we limit the 'self' to /content/ ?

File: src/subcommand/server.rs [hunk 2]
```diff
@@ -2285,7 +2285,7 @@ mod tests {
     server.assert_response_csp(
       format!("/preview/{}", InscriptionId::from(txid)),
       StatusCode::OK,
-      "default-src 'unsafe-eval' 'unsafe-inline' data:",
+      "default-src 'self' 'unsafe-eval' 'unsafe-inline' data:",
```
Comment (huuep) [inline src/subcommand/server.rs:2288]: do we need 'self' in /preview/?

File: src/subcommand/server.rs (hunk 1 above)
Comment (huuep) [reply src/subcommand/server.rs:826]: see this comment: https://github.com/ordinals/ord/issues/1082#issuecomment-1500268793

File: src/subcommand/server.rs (hunk 2 above)
Comment (ericatallah) [reply src/subcommand/server.rs:2288]: No, because `/preview/` loads the response from `/content/`

**Comment by clrke** [comment]
Closing PR because this has already been merged to ord

**Pull request closed without merging**